src/components/backend-ai-scaling-group-list.ts

```diff
@@ -576,6 +772,56 @@ export default class BackendAIScalingGroupList extends BackendAIPage {
             </mwc-button>
        </div>
       </backend-ai-dialog>
+      <backend-ai-dialog id="modify-scheduler-opts-dialog" fixed backdrop persistent>
+        <span slot="title">${_t('resourceGroup.SetSchedulerOptions')}</span>
+        <div slot="content" id="modify-scheduler-opts-container">
+          <mwc-select
+          id="allowed-session-types"
+          label="allowed_session_types"
+          required>
+            <mwc-list-item style="height:auto;" value="interactive">interactive</mwc-list-item>
+            <mwc-list-item style="height:auto;" value="batch">batch</mwc-list-item>
+            <mwc-list-item style="height:auto;" value="both">both</mwc-list-item>
+          </mwc-select>
+          <mwc-textfield
+            type="text"
+            value="0"
+            id="pending-timeout"
+            label="pending_timeout"
+            placeholder="0"
+            suffix="${_t('resourceGroup.TimeoutSeconds')}"
+            validationMessage="${_t('data.explorer.ValueRequired')}"
+            required
+            autoValidate
+            @input="${() => this._validatePendingTimeout()}"
+          ></mwc-textfield>
+        </div>
+        <div slot="footer" class="horizontal end-justified flex layout">
+          <mwc-button
+              icon="delete"
+              label="${_t('button.Refresh')}"
+              @click="${()=>this._clearValues()}"
+              style="width:100%"></mwc-button>
+          <mwc-button
+              unelevated
+              slot="footer"
+              icon="check"
+              label="${_t('button.Save')}"
+              @click="${()=>this.modifySchedulerOpts()}"
+              style="width:100%"></mwc-button>
+        </div>
+      </backend-ai-dialog>
+      <backend-ai-dialog id="show-detail-dialog" fixed backdrop persistent>
+        <span slot="title">${_t('agent.DetailedInformation')}</span>
+        <div slot="content" id="modify-scheduler-opts-container">
```

Comment:
How about using UI layout based on "keypair detail dialog"?
And please refer to key, value based UI Layout [here](https://design.mindsphere.io/patterns/key-value.html).

Reply:
Please check again about "keypair detail dialog" and update the UI according to that.